tomb

```diff
@@ -3335,7 +3335,6 @@ main() {
 	subcommands_opts[index]=""
 	subcommands_opts[search]=""
 
-	subcommands_opts[help]=""
```

Comment:
Duplicate of line [3330](https://github.com/dyne/Tomb/pull/511/files#diff-f2a6aa97adb2bf15c27aef69bf132106fbc7928a80f55475d0683daee82b4039L3330)

Reply:
This doesn't affect logic, but could be confusing for a developer.